### ISSUE TITLE
M5Stickc plus Display issues

### ISSUE THREAD

**Issue by macgyverfuerarme** [justcallmekoko/ESP32Marauder#281]
Please update the .bin file for m5stickc plus. On your discord is a fixed bin for the 10.5 version. The bin files here are displaying only a third of the screen. I wonder which .bin you used in your YouTube short. The menu in this short is much less smushed and looks more useful.

**Comment by f3shveg** [comment]
There are still some issues with screen after 0.10.8 update, i've seen a vertical orientation option in configs, i think this will help

**Comment by mik3l333** [comment]
[macgyverfuerarme](https://github.com/macgyverfuerarme)[f3shveg](https://github.com/f3shveg)
Have you been able to install it? Could you explain to me how for dummies? thank you

@macgyverfuerarme
@f3shveg

**Comment by f3shveg** [comment]
I think you using bin file from 0.10.8 release for M5StickC,  in that case you need to clone this git and open it as sketch, change everything you want to change in configs and then create bin file from sketch, I believe you can use same boot loader,partions you only need to change firmware to your new bin file
[Sketch to bin guide](https://randomnerdtutorials.com/bin-binary-files-sketch-arduino-ide/)

**Comment by mik3l333** [comment]
> I think you using bin file from 0.10.8 release for M5StickC, in that case you need to clone this git and open it as sketch, change everything you want to change in configs and then create bin file from sketch, I believe you can use same boot loader,partions you only need to change firmware to your new bin file [Sketch to bin guide](https://randomnerdtutorials.com/bin-binary-files-sketch-arduino-ide/)

66/70% of what you have told me sounds Chinese to me...
Could you explain it to me more simply? thank you.

**Comment by f3shveg** [comment]
Ok, in a few hours

**Comment by mik3l333** [comment]
> Ok, in a few hours

thank you so much my friend!

**Comment by f3shveg** [comment]
Ok,I’ve ran through config and I haven’t seen vertical option and ngl I’ve ran into some problems while compiling, I think I’ll open new threat, so what we have now is the best we can get

**Comment by laranja3** [comment]
Hello. 
I'm having the same problem.

I saw koko had some stuff here:
https://github.com/justcallmekoko/ESP32Marauder/commit/df613f3b9b8d4efc3671519075009bab2a6fbc39
I found it here:
https://github.com/justcallmekoko/ESP32Marauder/tree/df613f3b9b8d4efc3671519075009bab2a6fbc39

But idk if it helps...

I installed v0.10.8 here btw:
https://github.com/justcallmekoko/ESP32Marauder/releases/tag/v0.10.8

Do you know if i can just place the files from the 2 first links in https://esp.huhn.me/ and done?

Thank you!

**Comment by laranja3** [comment]
Hello.
My device won't turn on.
It has the right firmware.
It has full battery.
This has already happened to me before, when i first installed this firmware(the ESP32Marauder v0.10.8).However, after 30m it turned on and went well.
I tried some other stuff(other frimwares) for fun but then I deleted them from my device.
After it, i tried the M5burner with the title "M5StickCPlus-FactoryTest
official" and it ran smoothly.
Then, i deleted the firmware again.

NOW I'm trying to install the marauder firmware again (from here: https://github.com/justcallmekoko/ESP32Marauder/releases/tag/v0.10.8)
Thanks @macgyverfuerarme (;
But my device won't turn on!!!!!!!
Could someone explain to me how can i solve this?
